src/NUnitFramework/Fakes.cs

```diff
@@ -26,7 +26,7 @@
 using NUnit.Framework.Internal;
 using NUnit.Framework.Internal.Execution;
 
-#if PORTABLE && !NUNITLITE
+#if PORTABLE && !NUNITLITE && !NETSTANDARD1_6
```

Comment:
I understand they are standard, and the wiki should mention that, but I think we need a list of those defines we actually make use of. I've always found it annoying that we have to define the target runtime rather than having it done for us for by VS.